tests/Unit/Contrib/Prometheus/PrometheusExporterTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
just FYI, the current metrics implementation is about to be replaced, so this test will disappear with it.